inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -13,7 +13,7 @@
 
 
 def application_add(args: argparse.Namespace) -> str:
-    if (args.gpgKeyUri and args.gpgKeyName is None) or (args.gpgKeyName and args.gpgKeyUri is None):
+    if bool(args.gpgKeyUri) != bool(args.gpgKeyName):
         raise InbcException(
             "Source requires either both gpgKeyUri and gpgKeyName to be provided, or neither of them.")
```

Comment:
**Suggestion:** Consider using a more explicit conditional check for clarity. The current boolean conversion may not be immediately clear to all readers. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    if (args.gpgKeyUri and not args.gpgKeyName) or (args.gpgKeyName and not args.gpgKeyUri):
        raise InbcException(
            "Source requires either both gpgKeyUri and gpgKeyName to be provided, or neither of them.")
```